testcrate/tests/conv.rs

```diff
@@ -306,7 +306,11 @@ mod extend {
     }
 
     #[cfg(all(f16_enabled, f128_enabled))]
-    #[cfg(not(any(target_arch = "powerpc", target_arch = "powerpc64")))]
+    #[cfg(not(any(
+        target_arch = "powerpc",
+        target_arch = "powerpc64",
+        target_arch = "loongarch64"
+    )))]
```

Comment:
From the earlier failures, it looked like loongarch doesn't have/need __gnu_h2f_ieee and __gnu_f2h_ieee. I added a way to disable only these symbols, could you try adding loongarch here? https://github.com/rust-lang/compiler-builtins/blob/b7b93103fb9293c0c502dc1ae34e2ad5c871bc39/testcrate/build.rs#L95-L97